### PR TITLE
Sync admin flags between auth0 and czge

### PR DESCRIPTION
**Pull request by jgadling**
### Summary:
- **What:** Makes sure users invited with the admin role have that role reflected in our app.

### Checklist:
- [ ] I merged latest `<base branch>`
- [ ] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/aspen/api/schemas/usergroup.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class UserUpdateRequest(BaseRequest):
 class UserMeResponse(UserBaseResponse):
     split_id: str
     group: GroupResponse
+    group_admin: bool
```
Comment (jgadling) [inline src/backend/aspen/api/schemas/usergroup.py:46]: Adds group admin flag to the /users/me response @mmmmmmaya

File: src/backend/aspen/api/views/auth.py [hunk 2]
```diff
@@ -66,11 +68,14 @@ async def create_user_if_not_exists(db, userinfo):
         group = groupquery.scalars().one()  # type: ignore
     except NoResultFound:
         return  # We didn't find a matching group, we can't create this user.
+    # Get the user's roles for this organization and tag them as group admins if necessary.
+    roles = auth0_mgmt.get_org_user_roles(userinfo["org_id"], auth0_user_id)
```
Comment (jgadling) [inline src/backend/aspen/api/views/auth.py:72]: Get a list of roles associated with the auth0 user and group

File: src/backend/aspen/api/schemas/usergroup.py (hunk 1 above)
Comment (mmmmmmaya) [reply src/backend/aspen/api/schemas/usergroup.py:46]: Will this need updates when we get to multigroup, or will this stay the same structure and just update the value depending on which group is shown?

File: src/backend/aspen/api/views/groups.py [hunk 3]
```diff
@@ -134,7 +134,11 @@ async def invite_group_members(
         try:
             if success:
                 auth0_client.invite_member(
-                    organization["id"], client_id, user.name, email, "member"
+                    organization["id"],
+                    client_id,
+                    user.name,
+                    email,
+                    group_invitation_request.role,
```
Comment (jgadling) [inline src/backend/aspen/api/views/groups.py:141]: Send the role name that we get from the `group invite` cli to Auth0 so new users are created with a role.

**Review by lvreynoso — APPROVED** [review]
lgtm

File: src/backend/aspen/api/schemas/usergroup.py (hunk 1 above)
Comment (jgadling) [reply src/backend/aspen/api/schemas/usergroup.py:46]: hehe, I think that's a much bigger conversation -- let's talk next week!

**Review by mmmmmmaya — APPROVED** [review]

**Pull request merged**